child_compassion/models/field_office_disaster.py

```diff
@@ -0,0 +1,138 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck <mbcompte@gmail.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+
+from openerp import models, fields
+
+
+class ICPDisasterImpact(models.Model):
+    _name = 'icp.disaster.impact'
+    _description = 'ICP Disaster Impact'
+
+    compassion_project_id = fields.Many2one(
+        'res.country', 'Project'
```

Comment:
you can add attribute `ondelete='cascade'` to delete disaster impacts if project is deleted.